stubs/taint/core_privacy_security/django-filesystem-sinks.pysa

```diff
@@ -0,0 +1,12 @@
+def django.core.files.storage.FileSystemStorage.__init__(self, location: TaintSink[FileSystem_ReadWrite], base_url: TaintSink[FileSystem_ReadWrite], file_permissions_mode: TaintSink[FileSystem_ReadWrite], directory_permissions_mode: TaintSink[FileSystem_ReadWrite]): ...
+
+def django.core.files.storage.Storage.path(self, name: TaintSink[FileSystem_Other]): ...
+def django.core.files.storage.Storage.save(self, name: TaintSink[FileSystem_ReadWrite], content: TaintSink[FileSystem_ReadWrite], max_length): ...
+def django.core.files.storage.Storage.open(self, name: TaintSink[FileSystem_ReadWrite], mode): ...
+def django.core.files.storage.Storage.delete(self, name: TaintSink[FileSystem_Other]): ...
+def django.core.files.storage.Storage.generate_filename(self, filename: TaintSink[FileSystem_Other]): ...
```

Comment:
This function [doesn't actually perform](https://docs.djangoproject.com/en/1.11/ref/files/storage/#django.core.files.storage.Storage.generate_filename) any filesystem operations so the model can probably be deleted